unity/EditorPlugin/EditorPluginNet46.csproj

```diff
@@ -5,19 +5,20 @@
     <RootNamespace>JetBrains.Rider.Unity.Editor</RootNamespace>
     <DebugType>portable</DebugType>
     <DebugSymbols>true</DebugSymbols>
-    <DefineConstants>UNITY_2017_3;RIDER_EDITOR_PLUGIN</DefineConstants>
+    <DefineConstants>UNITY_2019_2;RIDER_EDITOR_PLUGIN</DefineConstants>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <AssemblyOriginatorKeyFile>..\..\sign.snk</AssemblyOriginatorKeyFile>
     <LangVersion>9</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="JetBrains.Unity.Libs.Ref.2017.3.0f3" Version="2020.6.10" />
+    <PackageReference Include="JetBrains.Unity.Libs.Ref.2019.2.0f1" Version="2021.11.29" />
```

Comment:
Does this break compatibility at all? What happens to Unity 2017.3 users?

Reply:
JetBrains.Rider.Unity.Editor.Plugin.Net46 is only used together with Rider package. So it can reference 2019.2+.
2019.2 has api, which we start using

Reply:
https://github.com/JetBrains/resharper-unity/pull/2244/files#diff-331358a28953ceb7298a8fd0123fa697ee04cd34d4a78e22dfd78ca5a8178108R21-R25